pkg/mover/chart.go

```diff
@@ -68,34 +68,51 @@ func (l *defaultLogger) Println(i ...interface{}) {
 	fmt.Println(i...)
 }
 
+// ChartMetadata exposes metadata about the Helm Chart to be relocated
+type ChartMetadata struct {
+	Name    string
+	Version string
+}
+
+// RegistryCredentials to access a registry
+type RegistryCredentials struct {
+	Username, Password string
```

Comment:
We'll likely need a server field, too.

Reply:
Do we? I mean if we map by host we might be repeating the host name in two places.

I will play with both approaches.

Reply:
Yes, you need to know what FQDN linked to the creds. And my understanding is that you can not authenticate to the same host with two different types of creds

Reply:
The current change has no use for a host also repeated inside the struct, just the host/FQDN defined as a map key in:
```
map[string]RegistryCredentials
```